Jv/ingestion examples (#14)
questdb/documentation: clients/java_ilp.md

import InterpolateReleaseData from "../../src/components/InterpolateReleaseData"
import { RemoteRepoExample } from "@theme/RemoteRepoExample"

> :::note
> 
> Note: This is the reference for the QuestDB Java Client when QuestDB is used as a server. For embedded QuestDB, please check our [Java Embedded Guide](/docs/reference/api/java-embedded/)
> 
> :::

Reviewer: ```suggestion
:::note

This is the reference for the QuestDB Java Client when QuestDB is used as a server. 

For embedded QuestDB, please check our [Java Embedded Guide](/docs/reference/api/java-embedded/).

:::
```

  `request_timeout`. This is useful for large requests. You can set this value
  to `0` to disable this logic.

### TLS encryption

> TLS in enabled by selecting the `https` or `tcps` protocol. The following
> options are available:

Reviewer: ```suggestion
To enable TLS, select the `https` or `tcps` protocol. 

The following options are available:
```
